blocks/additional-info/__list-item/additional-info__list-item.css

```diff
@@ -1,11 +1,11 @@
 .additional-info__list-item {
-  margin-bottom: 20px;
+  margin-bottom: 16px;
```

Comment:
You should add "last-of-type", you have extra margin on the second list item

Reply:
done